modules/luci-base/po/it/base.po

```diff
@@ -1016,19 +1018,19 @@ msgid "Enable HE.net dynamic endpoint update"
 msgstr "Abilitazione aggiornamento endpoint dinamico HE.net"
 
 msgid "Enable IPv6 negotiation"
-msgstr ""
+msgstr "Abilita negoziazione IPv6"
 
 msgid "Enable IPv6 negotiation on the PPP link"
 msgstr "Attiva la negoziazione IPv6 sul collegamento PPP"
 
 msgid "Enable Jumbo Frame passthrough"
 msgstr "Abilita Jumbo Frame passthrough"
 
-msgid "Enable NTP client"
+msgid "Enable client"
```

Comment:
I am pretty sure that this change is wrong.
You are changing the lookup string "Enable NTP client", and as there is no "Enable client" string, the translated string will be discarded.

Reply:
fixing it in 5 minutes